routing.rst

```diff
@@ -1336,15 +1336,19 @@ A possible solution is to change the parameter requirements to be more permissiv
 Route Aliasing
 --------------
 
-Route alias allow you to have multiple name for the same route:
+Route alias allows you to have multiple name for the same route
+and can be used to provide backward compatibility for routes that
+have been renamed.
```

Comment:
should we mention that this BC is only for route generation ? If you inspect the `_route` request attribute after url matching, the canonical name of the route will always be used here, not the name of an alias.

Reply:
👍 for me.

I think we could focus on this after complete update of Route Alias (including https://github.com/symfony/symfony-docs/pull/20638)?

We could address this topic individually on Route Alias and Deprecation paragraphs to indicate in each case what will be the value of `_route` requrest attribute. 

The current PR is more about clarification of the existing documentation.